src/components/Markers/SpotMarker.tsx

```diff
@@ -0,0 +1,9 @@
+import { Location } from "datasource";
+import React from "react";
+import { Marker } from "react-leaflet";
+
+type Props = Location;
```

Comment:
I'd prefer
```
type Props = {
 location: Location;
};
```

Location is a [ValueObject](https://www.martinfowler.com/bliki/ValueObject.html), it should be kept as a single unit, which would not be true if you added for example `onClick` to these props.